new.go

```diff
@@ -185,7 +184,7 @@ func resolveImage(ctx context.Context, systemContext *types.SystemContext, store
 		return nil, "", nil, fmt.Errorf("internal error: %d candidates (%#v) vs. %d failures (%#v)", len(candidates), candidates, len(failures), failures)
 	}
 
-	registriesConfPath := sysregistries.RegistriesConfPath(systemContext)
+	registriesConfPath := "registries.conf"
```

Comment:
Can't I get this via SystemContext.SystemRegistriesConfPath?

Reply:
@vrothberg @mtrmac ?

Reply:
I expect that field to not be set most of the time.